bin/preview-deploy/aws.user-data.sh

```diff
@@ -106,44 +112,48 @@ setsebool -P httpd_can_network_connect 1
 systemctl enable nginx
 systemctl restart nginx
 
+# Restart New Relic Infrastructure Monitor
+systemctl enable newrelic-infra
+systemctl start newrelic-infra
+
 # Setup pm2 to start itself at machine launch, and save its current
 # configuration to be restored when it starts
 su - ec2-user -c '~/.bash_profile; sudo env PATH=$PATH:/home/ec2-user/.nvm/versions/node/v14.16.0/bin /home/ec2-user/.nvm/versions/node/v14.16.0/lib/node_modules/pm2/bin/pm2 startup systemd -u ec2-user --hp /home/ec2-user'
 su - ec2-user -c 'pm2 save'
-su - ec2-user -c 'pm2 restart'
+su - ec2-user -c 'pm2 restart "eAPD API"'
 
 # These logs aren't created until after the Instance is provisioned and need to be added to CloudWatch as a last step
-touch /opt/aws/amazon-cloudwatch-agent/doc/launch-logs.json
-cat <<CWLAUNCHLOGCONFIG > /opt/aws/amazon-cloudwatch-agent/doc/launch-logs.json
+#touch /opt/aws/amazon-cloudwatch-agent/doc/launch-logs.json
```

Comment:
why is all of this commented out?

Reply:
I meant to test whether those logs were being captured, this is duplicate code. I believe there was a typo in one of the CloudWatch statements that I fixed, but I didn't test it and it's outside the scope of this work.